swagger2.0/oic.r.cloudopenapi.swagger.json

```diff
@@ -24,7 +27,7 @@
     }
   ],
   "paths": {
-    "/devices": {
+    "/api/v1/devices": {
```

Comment:
Why are you prepending /api/v1 to all the paths?  Doesn't this force the URI exposed by the Clouds to include this?

Reply:
Yes, that's the goal. It was like that also in the specification and also in the swagger, through the basePath parameter. But Dongjoo didn't know about the basePath parameter and others might be also confused, so I added it to each endpoint. It's based on the spec.

Reply:
This is different however to what you had before, resolving properly the basePath that you've now deleted you'd have:
/v1/devices
You've changed that to:
/api/v1/devices

I'm very leery of just putting this as an accepted change in the Swagger without having the TG ok this.

Reply:
In the spec we have`/api/v1`. It was just forgotten here, typo.